src/Stateless/StateRepresentation.cs

```diff
@@ -176,7 +176,7 @@ public void Enter(Transition transition, params object[] entryArgs)
                 }
             }
 
-            public void Exit(Transition transition)
+            public Transition Exit(Transition transition)
```

Comment:
Is there an `async` version of this method somewhere that we need to update?

Reply:
There's 'public async Task ExitAsync(Transition transition)'. I guess this has to modified to return Task<Transition>...